fmn/formatters.py

```diff
@@ -177,7 +176,45 @@ def irc_batch(messages, recipient):
     if len(messages) == 1:
         return irc(messages[0], recipient)
     else:
-        return fedmsg.meta.conglomerate(messages, **config.app_conf)
+        template = u'{subtitle} {delta} {link} {filter_link}\n'
+        formatted_message = u''
+        try:
+            # This apparently returns a list of dicts with the 'link', 'subtitle',
+            # 'subjective', 'secondary_icon' and any number of other keys from the
+            # individual conglomerator.
+            conglomerated = fedmsg.meta.conglomerate(messages, **config.app_conf)
+            for message in conglomerated:
+                formatted_message += template.format(
+                    subtitle=message['subtitle'], delta=message.get('human_time', ''),
+                    link=message['link'], filter_link=_irc_filter_link(recipient))
+        except Exception:
+            formatted_message = (
```

Comment:
I am not sure we should pass this directly to users... It's not like they are able to fix things.
Also, regardless of whether you send this to users, you really should also log them, preferably including traceback, so we can actually figure out what went wrong...

Reply:
Nice catch, logging statement added.

It's true there's not much they can do about it, but it's also going to be a weird set of messages and I just wanted them to know it's not normal. I'd welcome better phrasing, though.

Reply:
Maybe add a phrase like "This error has also been reported to the admins, but you can join #fedora-admin for more information"?
At least give users an idea of what to do with the info :).